app/config-default.js

```diff
@@ -6,6 +6,9 @@ module.exports = {
     // font family with optional fallbacks
     fontFamily: 'Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console", monospace',
 
+    // UI font family with optional fallbacks
+    uiFontFamily: '"-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif"',
+
```

Comment:
I think it's a bit too much for the default. Most people would expect system font to be the UI font and call it a day